.github/workflows/integration.yml

```diff
@@ -10,15 +10,22 @@ on:
     branches-ignore:
       - gh-pages
 
+env:
+  NODE_VER: '14.x'
+  JAVA_DISTRIBUTION: 'zulu'
+  JAVA_VER: 8
+
 jobs:
-  ubuntu:
+  mainbuild:
     name: ${{ matrix.os }} JDK 8
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
+        os: [ubuntu-20.04, ubuntu-22.04, macos-11, macos-12, windows-2019, windows-2022]
     runs-on: ${{ matrix.os }}
     env:
       NODE_VERSION: "lts/*"
```

Comment:
I'm also not too sure what it's for and I looked through the other .yml files they either set up this environment variable themselves or never reference it so I'm not too sure what it's for as well. It's present in the first version of this .yml file. Let's hope it's not some magic reference since I believe 'lts/*' is actually a way of stating 'use latest supported version'

Reply:
I think this NODE_VERSION doesn't mean anything because the `setup-node` action takes in another env variable for the Node version instead.

Reply:
yeah I noticed every where there is this lts line, there is separately a specific line that sets `node_version = (something)`.